gravitee-apim-console-webui/src/management/application/applications.route.ts

```diff
@@ -217,26 +216,38 @@ const applicationRoutes: Routes = [
         },
       },
       {
-        path: 'groups-ng',
+        path: 'groups',
         component: ApplicationGeneralGroupsComponent,
         data: {
           perms: {
-            only: ['application-definition-r'],
+            only: ['application-member-r'],
           },
           docs: {
-            page: 'management-application-groups',
+            page: 'management-application-members',
           },
         },
       },
       {
-        path: 'transfer-ownership-ng',
+        path: 'transfer-ownership',
         component: ApplicationGeneralTransferOwnershipComponent,
+        data: {
+          perms: {
+            only: ['application-member-r'],
+          },
+          docs: {
+            page: 'management-application-members',
+          },
+        },
+      },
+      {
+        path: 'general',
+        component: ApplicationGeneralNgComponent,
         data: {
           perms: {
             only: ['application-definition-r'],
           },
           docs: {
-            page: 'management-application-transferownership',
+            page: 'management-application-general-ng',
```

Comment:
why this new doc name ? 
WDYT to keep the existing one in previous routing : 
`page: 'management-application',`

Reply:
Done